AdaptiveUiCodelab/README.md

```diff
@@ -0,0 +1,22 @@
+# Adaptive and accessible apps in Jetpack Compose Codelab
+
+This folder contains the source code for
```

Comment:
can we mention that this is the solution?

Reply:
Sure let me add that and also update the readme headline as we changed the name of codelab